docs/network-iot/run-an-lns/configure-chirpstack.mdx

```diff
@@ -91,6 +92,70 @@ configuration example and documentation.
 
 Make sure ports 1700 and 1701 are both open on the server.
 
+
+### Example `chirpstack.toml` with Helium Additions.
+```toml
+# Network related configuration.
+[network]
+
+  # Network identifier (NetID, 3 bytes) encoded as HEX (e.g. 010203).
+  # new helium net_id="00003C"
+  net_id="00003C"
```

Comment:
I haven't gone through this setup myself. Does this work as `00003C` or does it need to be `0x00003C`?
Should update line 36 if it's wrong.

Reply:
yeh 00003C works